frontend/components/videoPlayer.vue

```diff
@@ -0,0 +1,46 @@
+<script setup>
+    let sourceBuffer;
+    //this is temporary just to pass the check, We will want this to be set dynamically
+    let codec = 'video/mp4; codecs="avc1.4D401F"';
```

Comment:
Should we have this in an env variable with an explanation on what it is? I see it is temporary, but I still don't understand what `avc1.4D401F` means. Will this only work on your computer with a given set of media? 
https://cconcolato.github.io/media-mime-support/

Reply:
We will want this to be set by the server through a request. We could do it through a http request or through the websocket, but this is the description of how the client is going to try to decode the video.  If this mismatches what the server sends at all it will give very mysterious errors and will be very hard to debug :)